src/components/AvatarList.astro

```diff
@@ -1,10 +1,56 @@
+---
+export let path;
+// fetch all commits for just this page's path
+let url = `https://api.github.com/repos/snowpackjs/astro-docs/commits?path=${path}`;
+let token = import.meta.env.SNOWPACK_PUBLIC_GITHUB_TOKEN ?? throw Error('You might not have "SNOWPACK_PUBLIC_GITHUB_TOKEN added for escaping rate-limiting.');
+let auth = `Basic ${Buffer.from(
+      token,
+      'binary'
+    ).toString('base64')}`;
+let data = await fetch(url, {
+  method: "GET",
+  headers: {
+    Authorization: auth,
+    "User-Agent": "astro-docs/1.0"
+  },
+}).then((res) => {
```

Comment:
astro supports top-level await! It would be nice to see this instead:
```
const res = await fetch(...
const data = await res.json();
```

Reply:
I can certainly do that, I just like scoped vars is all!

Reply:
I know, I always hate that await-fetch requires that unnecessary `res` var when all I want is the JSON